dashboard/app/dsl/external_dsl.rb

```diff
@@ -3,6 +3,8 @@ def initialize
     @hash = {href: '', options: {skip_dialog: true}}
   end
 
+  def associated_blocks(text) @hash[:associated_blocks] = text end
```

Comment:
is there a specific reason to do this all on one line?

Reply:
I'm not sure why that happened...